.classpath

```diff
@@ -37,7 +37,7 @@
 			<attribute name="module" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-12">
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER">
```

Comment:
I don't want this pushed up, as I'm switching to Java 12. If you get the latest Eclipse, there is an option to download the Java 12 support pack, which then enables Eclipse's support for it.

Reply:
Understood.   I tried everything to get the correct environment.

Reply:
Let me know if you are unable to find the right bits. I can dig up specific steps to take if so.